app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/eclair/EclairModels.scala

```diff
@@ -82,14 +102,23 @@ case class OpenChannelInfo(
     state: ChannelState.NORMAL.type
 ) extends ChannelInfo
 
+case class ActivatedFeature(feature: Feature, support: FeatureSupport)
+
+case class UnknownFeature(bitIndex: Int)
+
+case class Features(
+    activated: Set[ActivatedFeature],
+    unknown: Set[UnknownFeature])
+
 case class NodeInfo(
     signature: ECDigitalSignature,
-    features: String,
+    features: Features,
     timestamp: Instant,
     nodeId: NodeId,
     rgbColor: String,
     alias: String,
-    addresses: Vector[InetSocketAddress])
+    addresses: Vector[InetSocketAddress],
+    unknownFields: String)
```

Comment:
Isn't `unknownFields` in `features`?

Reply:
No, see https://acinq.github.io/eclair/#nodes